dynamicprefixvici.c

```diff
@@ -21,12 +30,44 @@ static char* getEnvironmentValue(const char* value);
 */
 static vici_req_t* createMessage(char* poolName, char* addressPool);
 
-int main(void)
+/**
+ * @brief get all command line arguments an passes gives it back in a struct.
+ * If unsupportet arguments are given, it terminates the program
+ * 
+ * @param argc 
+ * @param argv 
+ * @param arguments 
+ */
+static void ParseCommandLineArguments(int argc, char** argv, CommandLineArguments* arguments);
```

Comment:
Don't capitalize first word of variable

Reply:
Same for the function. 

https://w3.cs.jmu.edu/bernstdh/web/common/policies/styleguide-c.php#:~:text=Variable%20and%20method%2Ffunction%20names,start%20with%20an%20uppercase%20letter.

Variable and method/function names that contain multiple characters must not start with an uppercase letter.